src/ansys/aedt/core/filtersolutions_core/export_to_aedt.py

```diff
@@ -1221,12 +1221,12 @@ def interconnect_geometry_optimization_enabled(self, interconnect_geometry_optim
 
     def update_interconncet_parameters(self):
         """Update interconnect geometry equations with entered and selected parameters"""
-        status = self._dll.updateInterConnectParmeters()
+        status = self._dll.updateInterConnectParmeters()  # pragma: no cover
```

Comment:
Are we unable to test the two interconnect functions?

For the modelithics function, can we create a test for that which is skipped when modelithics is not installed?